test/model/unreads_test.dart

```diff
@@ -326,6 +326,48 @@ void main() {
           });
         }
       });
+
+      test('channelsWithUnreadMentions', () {
+        final stream1 = eg.stream();
+        final stream2 = eg.stream();
+
+        prepare();
+        fillWithMessages([
+          eg.streamMessage(stream: stream1, flags: [MessageFlag.mentioned]),
+          eg.streamMessage(stream: stream1, flags: []),
+          eg.streamMessage(stream: stream2, flags: []),
+        ]);
+
+        check(model.channelsWithUnreadMentions.single).equals(stream1.streamId);
```

Comment:
nit:
```suggestion
        check(model.channelsWithUnreadMentions).single.equals(stream1.streamId);
```
So that we get a better error message showing what's in `model.channelsWithUnreadMentions` if this fails.